demos/apps/bookinfo/bookinfo-istio.yaml

```diff
@@ -127,12 +127,14 @@ spec:
         ports:
         - containerPort: 9080
       - name: proxy
-        image: docker.io/istio/runtime:20170224214559
+        image: docker.io/istio/runtime:a87bea5
         securityContext:
           runAsUser: 1337
         args:
           - proxy
           - sidecar
+          - -n
```

Comment:
What is proxy is this yaml is deployed in a different namespace? Please remove this.